jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -961,18 +961,18 @@ ecma_builtin_array_prototype_object_sort_compare_helper (ecma_value_t j, /**< le
   {
     if (k_is_undef)
     {
-      *result_p = ecma_int32_to_number (0);
+      *result_p = ECMA_NUMBER_ZERO;
     }
     else
     {
-      *result_p = ecma_int32_to_number (1);
+      *result_p = ECMA_NUMBER_ONE;
     }
   }
   else
   {
     if (k_is_undef)
     {
-      *result_p = ecma_int32_to_number (-1);
+      *result_p = -ECMA_NUMBER_ONE;
```

Comment:
It might be a good idea to introduce a new macro for this (`ECMA_NUMBER_MINUS_ONE`).